bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/TargetFeedback.scala

```diff
@@ -146,9 +146,13 @@ object TargetGuidingFeedback {
   // star, show information particular to the guide star itself.
   def targetAnalysis(ctx: ObsContext, mt: MagnitudeTable, target: SPTarget): List[Row] = {
     val env = ctx.getTargets
-    if (target == env.getArbitraryTargetFromAsterism) baseAnalysis(ctx, mt)
-    else guideProbe(env, target).fold(List.empty[AnalysisRow]) { vgp =>
-      guideStarAnalysis(ctx, mt, vgp, target).toList
+    env.getSlewPositionObjectFromAsterism match {
+      case base: SPTarget if target == base =>
+        baseAnalysis(ctx, mt)
+      case _ =>
+        guideProbe(env, target).fold(List.empty[AnalysisRow]) { vgp =>
+          guideStarAnalysis(ctx, mt, vgp, target).toList
+        }
```

Comment:
I'm surprised that here we don't show the `baseAnalysis` for the two targets in the Ghost dual target mode case.  I would guess that the base analysis would be what you want to see unless you're parked on a guide star.

Reply:
I'll give this a little bit of thought.

Reply:
This is definitely an issue. I changed it so that no guiding feedback would be produced for any GHOST asterism targets, but we have the problem where if you click on a sky position or a group name node in the tree and you're missing a guide star, the warning isn't displayed.

I'll create a separate JIRA task for this since I think it falls outside the scope of this one.